roles/weave/tasks/main.yml

```diff
@@ -24,6 +25,10 @@
       dest: "/etc/systemd/system/weaveproxy.service"
   tags:
     - weave
+  notify:
+    - reload systemd
```

Comment:
is this going to cause some kind of weird effect? since in the ```reload systemd``` we have a notify for restarting weave/weaveproxy - but then we explicitly put it here too